src/referentielijsten/urls.py

```diff
@@ -48,8 +49,19 @@
         auth_views.PasswordResetCompleteView.as_view(),
         name="password_reset_complete",
     ),
+    path(
+        "api/",
+        include("referentielijsten.api.urls"),
+    ),
     # Simply show the master template.
-    path("", TemplateView.as_view(template_name="master.html"), name="root"),
+    path(
+        "",
+        TemplateView.as_view(
+            template_name="index.html",
+            extra_context={"version": api_settings.DEFAULT_VERSION},
+        ),
+    ),
+    path("ref/", include("vng_api_common.urls")),
```

Comment:
Sorry I don't get it. Could you please elaborate why we need them?

Reply:
We set the query param on required thus if it isn't given it will go though the vng_com
mon_api error detail view to render this. So without it we will encounter this error everytime we the user won't provide the required query param
![Screenshot from 2024-05-14 07-28-46](https://github.com/maykinmedia/referentielijsten/assets/101265650/c2c24b57-3d96-45da-a3c3-51252bd9a303)

Reply:
Oh I see, thanks for the explanation